edx_repo_tools/codemods/django3/add_new_django32_settings.py

```diff
@@ -0,0 +1,80 @@
+"""
+Codemod to add new settings in the repo settings file.
+"""
+import json
+import re
+import os
+import click
+from copy import deepcopy
+
+
+NEW_HASHING_ALGORITHM = "sha1"
+NEW_AUTO_FIELD = "django.db.models.AutoField"
+New_PROCESSOR = "django.template.context_processors.request"
+
+def update_hash_algorithm(file_data, new_algorithm=NEW_HASHING_ALGORITHM):
+    """
+    Update the HASHING_ALGORITHM in the file data and append if not found already.
+    """
+    algorithm_key = "DEFAULT_HASHING_ALGORITHM"
+    # if found the pattern, Update the algorithm value
+    if re.search(algorithm_key, file_data):
+        file_data = re.sub(
+            algorithm_key+" = .*\\n",
+            algorithm_key+f" = '{new_algorithm}'\n",
+            file_data
+        )
+    else:
+        # else, append the new algorithm at the end of the file
+        file_data = file_data + algorithm_key+f" = '{new_algorithm}'\n"
+    return deepcopy(file_data)
+
+def update_auto_field(file_data, new_field=NEW_AUTO_FIELD):
+    """
+    Update the AUTO_FIELD in the file and append if not found already.
+    """
+    field_key = "DEFAULT_AUTO_FIELD"
+    # if found the pattern, Update the field value
+    if re.search(field_key, file_data):
+        file_data = re.sub(
+            field_key+" = .*\\n",
```

Comment:
the same problem with this pattern too doesn't recognize (tried `DEFAULT_AUTO_FIELD = .*\\n`)
```
DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'
```

Reply:
Same here. The pattern works fine in case of reading the given string from the settings file but it doesn't matches the given string directly.